lasagne/layers/special.py

```diff
@@ -530,3 +534,102 @@ def prelu(layer, **kwargs):
     if nonlinearity is not None:
         layer.nonlinearity = nonlinearities.identity
     return ParametricRectifierLayer(layer, **kwargs)
+
+
+class RandomizedRectifierLayer(Layer):
+    """
+    lasagne.layers.RandomizedRectifierLayer(incoming,
+    lower=init.Constant(0.3), upper=init.Constant(0.8),
+    shared_axes='auto', **kwargs)
+
+    The randomized leaky rectifier was first proposed and used in the Kaggle
```

Comment:
The first sentence is used for the overview table in http://lasagne.readthedocs.org/en/latest/modules/layers.html, so it should be short and it must not include a reference (because that reference will not be defined where the table is rendered).